libs/sdk-backend-spi/src/workspace/execution/index.ts

```diff
@@ -456,4 +509,31 @@ export interface IDataView {
      * Thus, two data views on the same result, with same offset and limit will have the same fingerprint.
      */
     fingerprint(): string;
+
+    /**
+     * Calculate forecast data
+     * @beta
+     */
+    forecast(): IForecastView;
```

Comment:
This is not true - it does not calculate anything (forecast config + result is provided by `withForecast` method). Also what about if the forecast for this DataView was not provided? Should it throw, or return undefined?